packages/orbit-components/src/Timeline/TimelineStep/index.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+import React from "react";
+import styled, { css } from "styled-components";
+
+import defaultTheme from "../../defaultTheme";
+import Badge from "../../Badge";
+import Text from "../../Text";
+import Stack from "../../Stack";
+import CheckCircle from "../../icons/CheckCircle";
+import Circle from "../../icons/CircleEmpty";
+import AlertCircle from "../../icons/AlertCircle";
+import CloseCircle from "../../icons/CloseCircle";
+import { useStatuses, useStep } from "../TimelineContext";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+import type { Props } from "./index";
+
+const IconWrapperStyled = styled.div`
```

Comment:
As far most of our components are using unwritten naming taxonomy `StyledName` I prefer that you would stick to it and wouldn't have it reversed, so it code base is consistent 🙂